### ISSUE TITLE
Can't install via yarn

### ISSUE THREAD

**Issue by 19Gerhard85** [d-e-v-s-k/cuttr-js#9]
`$ yarn add cuttr --save`

```
yarn add v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
error /srv/project/node_modules/cuttr: Command failed.
Exit code: 1
Command: npx npm-force-resolutions
Arguments: 
Directory: /srv/project/node_modules/cuttr
Output:
npx: Installierte 6 in 2.477s
Error: ENOENT: no such file or directory, open './package-lock.json'
    at Object.openSync (fs.js:498:3)
    at Object.fs [as readFileSync] (fs.js:394:35)
    at npm_force_resolutions$core$node_slurp (/home/user/.npm/_npx/43287/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.cljs:15:20)
    at npm_force_resolutions$core$read_json (/home/user/.npm/_npx/43287/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.cljs:22:23)
    at switch__2144__auto__ (/home/user/.npm/_npx/43287/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.cljs:151:3)
    at /home/user/.npm/_npx/43287/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.cljs:151:3
    at npm_force_resolutions$core$update_package_lock_$_state_machine__2145__auto____1 (/home/user/.npm/_npx/43287/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.js:648:4)
    at cljs.core.async.impl.ioc-helpers/FN-IDX (/home/user/.npm/_npx/43287/lib/node_modules/npm-force-resolutions/out/npm_force_resolutions/core.js:664:88)
    at cljs.core.async.impl.ioc-helpers/run-state-machine (/home/user/.npm/_npx/43287/lib/node_modules/npm-force-resolutions/out/cljs/core/async/impl/ioc_helpers.cljs:35:23)
    at cljs$core$async$impl$ioc_helpers$run_state_machine_wrapped (/home/user/.npm/_npx/43287/lib/node_modules/npm-force-resolutions/out/
```

**yarn version: 1.22.19**

**Comment by d-e-v-s-k** [comment]
Hi @19Gerhard85 

it was an issue regarding the missing package-lock file.

This one should be solved with [Cuttr v1.4.3](https://github.com/d-e-v-s-k/cuttr-js/releases/tag/1.4.3) - please give it a try.

**Comment by 19Gerhard85** [comment]
@d-e-v-s-k 
Yup, works now, thanks!
